app/assets/stylesheets/components/_container.sass

```diff
@@ -1,4 +1,4 @@
-$Container-maxWidth: 960px
+$Container-maxWidth: 1440px
```

Comment:
@fcbaila @johnny1337 This is the max-width for the content. Isn't this too much?